modules/scheduler/executor/plugins/k8s/namespace/namespace.go

```diff
@@ -71,7 +71,7 @@ func (n *Namespace) Create(ns string, labels map[string]string) error {
 	if !resp.IsOK() {
 		return errors.Errorf("failed to create namespace, ns: %s, statuscode: %v, body: %v", ns, resp.StatusCode(), b.String())
 	}
-	logrus.Infof("succeed to create namespace, ns: %s", ns)
+	logrus.Infof("succeed to create namespace, ns %s from module %s", ns, module)
```

Comment:
parameters should not only be used to print logs.
pls delete this param (module).

Reply:
already  modify to 
```suggestion
logrus.Infof("succeed to create namespace, ns: %s", ns)
```